app/models/wpcc/config_loader.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  class ConfigLoader
+    def self.load(filename)
+      hash = YAML.load_file(Wpcc::Engine.root.join('config', "#{filename}.yml"))
```

Comment:
So what happens if the file does not exist?

Reply:
Why will you try to load a file that don't exist?

Reply:
Mistakes can happen. If you don't think human error can innocently screw things up then you can go ahead and commit this.

Reply:
I agree with mistakes can happen. But tries to change a constant to a file which don't exist and see what happens.

Reply:
And if you try to run the tests, see what exactly happen as well.